frontend/src/employee-frontend/components/child-information/assistance-need/decision/AssistanceNeedDecisionEditPage.tsx

```diff
@@ -26,17 +27,26 @@ import {
   FixedSpaceColumn,
   FixedSpaceRow
 } from 'lib-components/layout/flex-helpers'
+import { AlertBox } from 'lib-components/molecules/MessageBoxes'
 import Select, { SelectOption } from 'lib-components/molecules/Select'
-import { H1, H2, Label } from 'lib-components/typography'
+import { H1, H2, Label, P } from 'lib-components/typography'
 import { Gap } from 'lib-components/white-space'
 import { featureFlags } from 'lib-customizations/employee'
 
 import { getPerson } from '../../../../api/person'
 
-import AssistanceNeededDecisionForm from './AssistanceNeededDecisionForm'
+import AssistanceNeededDecisionForm, {
+  FieldInfos
+} from './AssistanceNeededDecisionForm'
 import { useAssistanceNeedDecision } from './assistance-need-decision-form'
 import { FooterContainer, DecisionInfoHeader } from './common'
 
+const requiredFormFields = ['selectedUnit' as const]
```

Comment:
You should mark the whole array with `as const` so that every additional element doesn't have to be marked with `as const` as well.